keyboards/dztech/dz60v2/readme.md

```diff
@@ -0,0 +1,29 @@
+# DZ60 60% Mechanical Keyboard PCB
+
+## A Customizable 60% Mechanical Keyboard PCB
+
+![A Customizable 60% Mechanical Keyboard PCB](https://cdn.shopify.com/s/files/1/1473/3902/files/1_03_abb48769-9486-44bd-a761-7cd39d7e1bda.jpg)
```

Comment:
Please rehost on a dedicated image hosting site (Imgur, etc).  Shopify removes the images, so this may end up as a dead link very easily.